src/main/java/net/coderbot/iris/shaderpack/ShaderPack.java

```diff
@@ -18,27 +18,24 @@
 
 public class ShaderPack {
 	private final ProgramSet base;
-	@Nullable
-	private final ProgramSet overworld;
-	private final ProgramSet nether;
-	private final ProgramSet end;
 
 	private final IdMap idMap;
 	private final Map<String, Map<String, String>> langMap;
 	private final CustomTexture customNoiseTexture;
 
+	private final ShaderProperties shaderProperties;
+	private final Path root;
+
 	public ShaderPack(Path root) throws IOException {
 		// A null path is not allowed.
 		Objects.requireNonNull(root);
 
-		ShaderProperties shaderProperties = loadProperties(root, "shaders.properties")
+		this.shaderProperties = loadProperties(root, "shaders.properties")
```

Comment:
ShaderProperties is a temporary intermediate that isn't intended to be stored long-term.